geosimple/fields.py

```diff
@@ -4,8 +4,6 @@
 
 class GeohashField(models.CharField):
 
```

Comment:
What does removing this do? This stack overflow question seems to indicate we might need to override the from_db_value function? https://stackoverflow.com/questions/35166085/how-to-deal-with-subfieldbase-has-been-deprecated-use-field-from-db-value-inst

Reply:
surprisingly the original developer of geosimple keeps their project in track, but in general I would not recommend using less supported external dependencies
will just Ctrl+V their Django 1.10 support :)